File: packages/sdk-typescript/examples/feedback-trust-workflow.ts
import { ArcEnCielClient } from '../src/index.js'

const apiKey = process.env.ARCENCIEL_API_KEY
if (!apiKey) throw new Error('Set ARCENCIEL_API_KEY with FeedbackRead for this read-only example.')

const client = new ArcEnCielClient({ apiKey })
const feedback = await client.call(() => client.feedback.listMyFeedback())
const notices = await client.call(() => client.trustSafety.listMyIllegalContentNotices())

console.log(`Own feedback cases: ${feedback.length}`)
console.log(`Own illegal-content notices: ${notices.length}`)

if (feedback[0]) {
  const detail = await client.call(() => client.feedback.getFeedback({ publicId: feedback[0].publicId }))
  console.log(`${detail.publicId}: ${detail.status} — ${detail.title}`)
}

if (notices[0]) {
  const detail = await client.call(() => client.trustSafety.getIllegalContentNotice({ publicId: notices[0].publicId }))
  console.log(`${detail.publicId}: ${detail.status} — ${detail.targetType}`)
}

File: packages/sdk-python/examples/feedback_trust_workflow.py
"""Read-only Developer API v1.8 feedback and Trust & Safety smoke workflow."""

from __future__ import annotations

import os

from arcenciel import ArcEnCielClient

api_key = os.environ.get("ARCENCIEL_API_KEY")
if not api_key:
    raise RuntimeError("Set ARCENCIEL_API_KEY with FeedbackRead for this read-only example.")

client = ArcEnCielClient(api_key=api_key)
feedback = client.call_sync(client.feedback.list_my_feedback_sync)
notices = client.call_sync(client.trust_safety.list_my_illegal_content_notices_sync)

print(f"Own feedback cases: {len(feedback)}")
print(f"Own illegal-content notices: {len(notices)}")

if feedback:
    detail = client.call_sync(
        lambda: client.feedback.get_feedback_sync(public_id=feedback[0].public_id)
    )
    print(f"{detail.public_id}: {detail.status} — {detail.title}")

if notices:
    detail = client.call_sync(
        lambda: client.trust_safety.get_illegal_content_notice_sync(public_id=notices[0].public_id)
    )
    print(f"{detail.public_id}: {detail.status} — {detail.target_type}")
